doc/manual/src/design/store/related-work.md

```diff
@@ -0,0 +1,37 @@
+# Advanced Topic: Related Work
```

Comment:
I don't think this is helpful right now. Let's try to keep new material to a minimum, and collect tangential ideas in private. It's hard to keep track of all that stuff in public, and produces a lot of noise.